netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1404,8 +1365,7 @@ net_ebpf_extension_sock_addr_redirect_connection_classify(
     }
     if (!net_ebpf_extension_hook_client_enter_rundown(attached_client)) {
         attached_client = NULL;
-        // Client is detaching, change action to permit.
-        action = FWP_ACTION_PERMIT;
+        // Client is detaching.
         goto Exit;
```

Comment:
This seems to be a significant action, so we might want to add a log message here as well?  On a more general note, given that this function is a callback invoked by the system, it might be a good idea to add generous logging.